onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -4666,11 +4666,11 @@ def aten_le_bool(self: BOOL, other: BOOL) -> BOOL:
 def aten_lerp(self: TReal, end: TReal, weight: TReal) -> TReal:
```

Comment:
If cast is needed, weight can take a different type than self

Reply:
Fixed, I was doing the casting incorrectly.